addOns/graphql/src/test/java/org/zaproxy/addon/graphql/GraphQlFingerprinterUnitTest.java

```diff
@@ -385,6 +386,14 @@ void shouldFingerprintAfterHandlerReset() throws Exception {
         assertNoErrors(extensionAlert, writer.toString());
     }
 
+    @Test
+    void shouldStaticallyAddHandlerWithoutException() throws Exception {
```

Comment:
I'd suggest setting an order for this one, to ensure that this is called first (the case when this NPE would have happened).

Reply:
Done